### PR TITLE
Daniel/csv

### PR DESCRIPTION
**Pull request by alined908**
- upload csv of participants
- export csv of matching
- cuke tests
- page title chore
- participants/matching css + refactoring

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -18,5 +18,12 @@
 //= require bootstrap
 //= require bootstrap-datepicker
 //= require_tree .
+/* global $ */
+
+/*Change label to be name of uploaded file */
+$(document).on('change', '.custom-file-input', function(){
```
Comment (kev-chien) [inline app/assets/javascripts/application.js:24]: Interesting way to solve the problem, looks good

File: features/step_definitions/participants_steps.rb [hunk 2]
```diff
@@ -29,3 +29,11 @@
     participant = Participant.find(part_id)
     visit "/projects/"+ project_id + "/participants/" + part_id + "/ranking/edit?secret_id=" + participant.secret_id + "a"
 end
+
+When /^I upload a non csv file$/ do
+    attach_file(File.join(RAILS_ROOT, 'features', 'attach_file', 'dog.png'))
```
Comment (kev-chien) [inline features/step_definitions/participants_steps.rb:34]: ka yoot

**Review by Turese — APPROVED** [review]
Looks good. he changed labels on the participants page are better and more indicative of the functions of the different commands. The tests look good and cover the csv cases properly. The UI for adding csv files looks neat. It might benefit from the ability to download csvs from the participants page as well, instead of just downloading csvs of the entire project on the matching page, but that's a minor nitpick.

File: app/views/participants/display.html.erb [hunk 3]
```diff
@@ -22,15 +25,15 @@
           <td><%= participant.email %></td>
           <td><%= participant.last_responded ? time_ago_in_words(participant.last_responded) + ' ago' : 'No response yet'%></td>
           <td>
-            <%= link_to 'Autofill Rankings', autofill_project_participant_path(participant.project_id, participant.id), class:"btn btn-primary"%>
-            <%= link_to 'DELETE', project_participant_path(participant.project_id, participant.id), method: :delete, url: project_participants_path, class:"btn btn-danger"%>
-            <%= link_to 'Show Rankings', project_participant_ranking_path(participant.project_id, participant.id), class:"btn btn-success" %>
+            <%= link_to 'Autofill Rankings', autofill_project_participant_path(participant.project_id, participant.id), class:"btn btn-primary btn-sm"%>
+            <%= link_to 'Delete', project_participant_path(participant.project_id, participant.id), method: :delete, url: project_participants_path, class:"btn btn-danger btn-sm"%>
```
Comment (kev-chien) [inline app/views/participants/display.html.erb:29]: Appreciate this, it looks nicer this way.

**Review by kev-chien — APPROVED** [review]
Looks good! Thanks for making the tests so comprehensive! It covers many of the edge cases and different use scenarios.
